epochStart/bootstrap/bootStrapSovereignShardProcessor.go

```diff
@@ -0,0 +1,209 @@
+package bootstrap
+
+import (
+	"context"
+	"time"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-core-go/data"
+	"github.com/multiversx/mx-chain-core-go/data/block"
+	"github.com/multiversx/mx-chain-core-go/data/typeConverters/uint64ByteSlice"
+	"github.com/multiversx/mx-chain-go/dataRetriever/factory/containers"
+	requesterscontainer "github.com/multiversx/mx-chain-go/dataRetriever/factory/requestersContainer"
+	"github.com/multiversx/mx-chain-go/dataRetriever/requestHandlers"
+	"github.com/multiversx/mx-chain-go/epochStart/bootstrap/disabled"
+	"github.com/multiversx/mx-chain-go/process"
+	"github.com/multiversx/mx-chain-go/sharding/nodesCoordinator"
+	"github.com/multiversx/mx-chain-go/storage/cache"
+	"github.com/multiversx/mx-chain-go/trie/factory"
+)
+
+type sovereignBootStrapShardProcessor struct {
+	*sovereignChainEpochStartBootstrap
+}
+
+func (e *sovereignBootStrapShardProcessor) requestAndProcessForShard(peerMiniBlocks []*block.MiniBlock) error {
```

Comment:
the "e" is wrong as sovereignBootstrapShardProcessor does not start with e. it is hard to read. Change to to "s" maybe.

Reply:
Right, good point.
I'll change it in the last PR, since this would cause conflicts for each merged PR which are on top of this one.

Last PR will contain all these changes